README.md

```diff
@@ -1,2 +1,219 @@
+- Open http://localhost:3000 to view it in the browser.
+- The page will reload if you make edits.
+- You will also see any lint errors in the console.
+- Run npm run build to build the app for production to the build folder.
+- It correctly bundles React in production mode and optimizes the build for the best performance.
+- The build is minified and the filenames include the hashes.
+- Your app is ready to be deployed!
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have visual studio Code in your PC.
+- Have a verified GitHub account.
+- A code editor
+- Nodejs
+- NPM
+- Have a stable browser version.
+
+### Setup
+
+- Create a local directory where you can clone the project.
+- Clone this repository to your desired folder using terminal: 
+git clone https://github.com/ReemMohamedAbdelfatah/air-pollution-app.git
+- Install the dependencies
+- Open the project on a browser
+
+### Install
+
+Install this project with:
+
+- Just run npm i to install all dependencies
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm run build
+- npm run start
+
+### Run tests
+
+To run tests, run the following command:
+
+Coming Soon ....
+
+### Deployment
+
+You can deploy this project using:
+
+Coming Soon ....
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Reem Muhammed**
+
+- GitHub: [@github](https://github.com/ReemMohamedAbdelfatah)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/reem-muhammad-a07543116/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Better_design]**
+- [ ] **[Add_map_foreach_country]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ReemMohamedAbdelfatah/air-pollution-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please star the project and feel free to add your own touches.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank microverse for the great resources shared. Also I would like to thanks Nelson Sakwa for Original Design Official Behance Page @NelsonSakwa, Api - Open Weather
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+
+- **[How_to_use_the_app?]**
+
+  - [I puted a small div in the top explaining how to use tha application]
+
+- **[Can_I_Contribute?]**
+
+  - [Sure, you're free to contribute and add your own touches]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE/MIT.md) licensed.
```

Comment:
- [ ] MIT license link in your ReadMe is broken, kindly endeavor to fix it.

Reply:
Done 👌